packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -25,31 +20,58 @@ import { InputText } from '../../components/InputText';
 import LinkRememberPassword from '../../components/LinkRememberPassword';
 import WarningMessage from '../../components/WarningMessage';
 import { isEmailValid } from 'fxa-shared/email/helpers';
-import sentryMetrics from 'fxa-shared/lib/sentry';
+import { ResetPasswordQueryParams } from '../../models/reset-password';
 
 export const viewName = 'reset-password';
 
 export type ResetPasswordProps = {
   prefillEmail?: string;
   forceAuth?: boolean;
   serviceName?: MozServices;
+  beginResetPasswordHandler: BeginResetPasswordHandler;
+  queryParams: ResetPasswordQueryParams;
 };
 
+export interface PasswordForgotSendCodeResponse {
+  passwordForgotSendCode: {
+    passwordForgotToken: string;
+  };
+}
+
+export interface BeginResetPasswordResult {
+  data?: PasswordForgotSendCodeResponse | null;
```

Comment:
If I push another update I'll remove `| null` here, don't think we need it after latest update.